modules/packer/custom-image/README.md

```diff
@@ -30,6 +30,29 @@ images to internal projects.
 [hpcimage]: https://cloud.google.com/compute/docs/instances/create-hpc-vm
 [startup-metadata]: https://cloud.google.com/compute/docs/instances/startup-scripts/linux
 
+## Example blueprints
+
+Several examples blueprints that create a startup-scripts using the HPC Toolkit [startup-script] module and then use it to build a specialized image can be found below.
```

Comment:
nit: This sentence feels long and hard to follow. I would suggest:

> A recommended pattern for building images with this module is to use the terraform based [startup-script] module along with this packer `custom-image` module. Below you can find links to several examples of this pattern, including usage instructions.

Reply:
PTAL